awx/main/migrations/_create_system_jobs.py

```diff
@@ -15,9 +16,41 @@
 only if new system job templates were created (i.e. new database).
 '''
 
+def create_collection_jt(apps, schema_editor):	
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')	
+    Schedule = apps.get_model('main', 'Schedule')	
+    ContentType = apps.get_model('contenttypes', 'ContentType')	
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)	
+    now_dt = now()	
+    random_time = now_dt + timedelta(minutes=random.randint(-30,30))	
+    random_schedule_time = random_time.strftime('%Y%m%dT%H%M%SZ')
+    insights_enabled = getattr(settings, 'INSIGHTS_TRACKING_STATE', False)
 
-def create_clearsessions_jt(apps, schema_editor):
+    sjt, created = SystemJobTemplate.objects.get_or_create(	
+        job_type='gather_analytics --ship',
```

Comment:
Is this `--ship` what we want here?  This doesn't seem to match the type options in defined at the migration above.